msteams-platform/samples/app-templates.md

```diff
@@ -23,6 +23,18 @@ All app templates are provided under the [MIT License](https://github.com/Office
 * **Customizable and extensible:** While all app templates can be deployed with minimal configuration, we provide the entire code base and deployment scripts so that you can easily customize or extend them to fit your unique needs.
 * **Detailed documentation:** All app templates are accompanied by end-to-end documentation on solution architecture, deployment, and configuration steps.  
 
+## Adoption tool - Champion Management Platform &#9734;
+
+The Champion Management Platform (CMP) app template helps you manage, scale, and inspire your teamwork champions to achieve more. This app template is built on the SharePoint Framework and loaded into a tab within a team. Groups can leverage this tool to help manage program membership, provide a leaderboard and event types for logging, as well as tooling to overlay digital badges to program participants.
```

Comment:
The sentence can be written as:

Groups can leverage this tool and manage program membership, provide a leaderboard and event types for logging, and tools to overlay digital badges to program participants.